### PR TITLE
fix:issues #246

### PR DESCRIPTION
**Pull request by MapleWithoutWords**

### CODE REVIEW

**Comment by MapleWithoutWords** [comment]
#246  需要等待 https://github.com/masastack/MASA.TSC/pull/305 合并并发布

File: src/Services/Masa.Scheduler.Services.Worker/Extensions/WebApplicationBuilderExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+using Masa.BuildingBlocks.Data;
```
Comment (wzh425) [inline src/Services/Masa.Scheduler.Services.Worker/Extensions/WebApplicationBuilderExtensions.cs:4]: global using

File: src/Services/Masa.Scheduler.Services.Worker/Program.cs [hunk 2]
```diff
@@ -6,20 +6,18 @@
...
-        ServiceVersion = masaStackConfig.Version,
-        ServiceName = masaStackConfig.GetServerId(MasaStackConstant.SCHEDULER, "worker"),
-        Layer = masaStackConfig.Namespace,
-        ServiceInstanceId = builder.Configuration.GetValue<string>("HOSTNAME")
-    };
-}, () =>
-{
-    return masaStackConfig.OtlpUrl;
-});
+//builder.Services.AddObservable(builder.Logging, () =>
```
Comment (wzh425) [inline src/Services/Masa.Scheduler.Services.Worker/Program.cs:9]: Remove Comment

**Review by wzh425 — COMMENTED** [review]
@MapleWithoutWords

**Review by wzh425 — APPROVED** [review]

**Pull request merged**